experiment/maintenance/README.md

```diff
@@ -0,0 +1,3 @@
+## experiment/maintenance
```

Comment:
hummm... just put this in `maintenance/`?

Reply:
these scripts are rarely used and *barely* belong here at all, they're certainly not in production.